### PR TITLE
MM-50078-unable-to-click-the-create-button-after-fixing-the-name-of-the-work-template

### PR DESCRIPTION
**Pull request by tiagodll**
#### Summary
It clears the error message to unblock the create button.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-50078

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/work_templates/index.tsx [hunk 1]
```diff
@@ -157,6 +157,7 @@ const WorkTemplateModal = () => {
 
     const handleOnNameChanged = (name: string) => {
         trackEvent(TELEMETRY_CATEGORIES.WORK_TEMPLATES, 'customize_name', {category: selectedTemplate?.category, template: selectedTemplate?.id});
+        setErrorText('');
```
Comment (julmondragon) [inline components/work_templates/index.tsx:160]: nit: For consistency on the effect being used to reset error, could instead add `selectedName` as dependency:
https://github.com/mattermost/mattermost-webapp/blob/4635e9152ecc5ff46fca55bc5933494c77778299/components/work_templates/index.tsx#L119-L122

**Review by julmondragon — APPROVED** [review]
LGTM @tiagodll

File: components/work_templates/index.tsx (hunk 1 above)
Comment (JulienTant) [reply components/work_templates/index.tsx:160]: +1 with @julmondragon but not nit from me :P

Comment (tiagodll) [reply components/work_templates/index.tsx]: thanks for the suggestion
done

**Review by JulienTant — APPROVED** [review]
Thanks @tiagodll!

**Comment by stevemudie** [comment]
I just tested this and while it worked as expected based on the QA steps in [the ticket](https://mattermost.atlassian.net/browse/MM-50078) but theres more to it.

- The error appears as expected but there's no indication to the user as to what caused the error or how to fix it @annelieseklein 
- You can use hundreds of characters if you'd lie (well beyond the 64 character limit) and only need to delete a single letter to re-enable the `Create` button - which will just lead you to the same error when you click it again
Screenshot 2023-02-08 at 10 01 21 AM

**Comment by JulienTant** [comment]
Hey @stevemudie ; those issues will be addressed at part of https://mattermost.atlassian.net/browse/MM-50080

**Review by stevemudie — APPROVED** [review]

**Comment by JulienTant** [comment]
Merging this, thanks @tiagodll <3

**Pull request merged**